wikitextprocessor/core.py

```diff
@@ -1692,37 +1692,46 @@ def get_page(
         title = title.replace("_", " ")
         if title.startswith("Main:"):
             title = title[5:]
+
+        lower_case_title = title
         if namespace_id is not None and namespace_id != 0:
             local_ns_name = self.LOCAL_NS_NAME_BY_ID[namespace_id]
             ns_prefix = local_ns_name + ":"
-            if self.lang_code == "zh" and namespace_id in {
+            if namespace_id in {
                 self.NAMESPACE_DATA[ns]["id"] for ns in ["Template", "Module"]
             }:
-                # Chinese Wiktionary capitalizes the first letter of template/module
-                # page titles but uses lower case in Wikitext and Lua code
+                # Chinese Wiktionary and English Wikipedia capitalizes the first
+                # letter of template/module page titles but uses lower case in
+                # Wikitext and Lua code
                 if title.startswith(ns_prefix):
                     template_name = title[len(ns_prefix) :]
-                    title = (
+                    lower_case_title = (
                         ns_prefix + template_name[0].upper() + template_name[1:]
                     )
                 else:
-                    title = ns_prefix + title[0].upper() + title[1:]
+                    lower_case_title = ns_prefix + title[0].upper() + title[1:]
```

Comment:
If you use .upper() here, then `lower_case_title` has the wrong name; it's an upper_case_title.

Reply:
Thanks! I don't know why I named it `lower_case`...